ansible/playbooks/etcd-setup.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
As word ``setup`` is very close to work ``configure``, I would use ``<role>-deploy`` instead the setup form. At the same time ``<action>-<role>`` sounds more friendly than ``<role>-setup``. As we have ``deploy-cluster`` already, let's align with that.